changelog.md

```diff
@@ -305,6 +305,15 @@
 - Added `iterable[T]` type class to match called iterators, which enables writing:
   `template fn(a: iterable)` instead of `template fn(a: untyped)`
 
+- A new import syntax `import {.all} foo` now allows to import all symbols (public or private)
```

Comment:
Nit, in consistency with tests and how pragmas are usually written:
```suggestion
- A new import syntax `import {.all.} foo` now allows to import all symbols (public or private)
```

Reply:
done